config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
-    catalogUrl: null,
+    catalogUrl: "https://stac-api.test.software.internal",
```

Comment:
```suggestion
    catalogUrl: null,
```

I would suggest to keep it `null` so it fails if it is not provided